index.html

```diff
@@ -64,6 +64,37 @@ <h2 class="headline-section__sub-title">
 		<!-- WORK -->
 		<section class="work-section" id="portfolio">
 			<div class="work-section__box">
+				<div class="pop-up--container" id="con-1">
+					<div class="pop-up__header">
+						<h3 class="project-name">Tonic</h3>
+						<div class="close-btn"><img src="./images/x.png" alt="close"></div>
+					</div>
+					<div class="range">
+						<p class="range__canopy">CANOPY</p>
+						<div class="range__circle"></div>
+						<p class="range__txt">Back End Dev</p>
+						<div class="range__circle"></div>
+						<p class="range__txt">2015</p>
+					</div>
+					<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio.png" alt="card snap"></div>
+				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop1.png" alt="card snap"></div>
+				<div class="pop-up__footer">
+					<p class="desc">
+						Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release.
+				</p>
+				<div class="options">
+					<ul class="tech">
+						<li class="tech__type">html</li>
+						<li class="tech__type">css</li>
+						<li class="tech__type">javaScript</li>
+					</ul>
+					<div class="buttons-action">
+						<button type="button" class="btn see-project">See live <img src="./images/frame2.png" alt="icon"></button>
+						<button type="button" class="btn see-project">See Source <img src="./images/Frame1.svg" alt="icon"></button>
+					</div>
+				</div>
+				</div>
+				</div>
```

Comment:
- [ ] Your pop-up is greatly implemented, however, according to the project requirements:

> ![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)
> 
> As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.
> 
> This means you can use the `projects` array and logic that you already have, to populate the projects section of the main page entirely with javascript!
> 
> If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)
> 





- [ ]  All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty